File: test/csharp/SpecGate.CtscFixtures/Conformance/MultiFile/Farewell.cs
using SpecGate.Annotations;

namespace SpecGate.CtscFixtures.Conformance.MultiFile;

public static class Farewell
{
    [SpecOperation("farewell", Spec = "fixture.multi_file")]
    public static string Execute([SpecInput("name")] string name) => $"Goodbye, {name}!";
}

File: test/rust/crates/specgate-ctsc-fixtures/src/conformance/multi_file/farewell.rs
use specgate::spec_operation;

#[spec_operation("farewell", spec = "fixture.multi_file")]
pub fn farewell(name: &str) -> String {
    format!("Goodbye, {name}!")
}

#[test]
fn farewells_from_second_source_file() {
    assert_eq!(farewell("Ada"), "Goodbye, Ada!");
}
